### PR TITLE
18414 fix focus lost on almost any action

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#18414` fix focus lost on almost any action`](http://community.openproject.org/work_packages/18414)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-fix-18414-fix-focus-lost-on-special-case.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-fix-18414-fix-focus-lost-on-special-case.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by floriank** [comment]
I can verify that this now works on Firefox, Chrome and Opera. @NobodysNightmare could you have a look in IE?

**Comment by NobodysNightmare** [comment]
LGTM 4 IE11

**tl; dr:** :+1:
